src/glossary.rst

```diff
@@ -1489,7 +1493,6 @@ Glossary
 
 
    required output
-   expected output
```

Comment:
Term "expected output" never used elsewhere, simpler to remove this unused term and stick to "required".